WordPressKit/Decodable+Dictionary.swift

```diff
@@ -0,0 +1,100 @@
+struct JSONCodingKeys: CodingKey {
+    var stringValue: String
+
+    init?(stringValue: String) {
+        self.stringValue = stringValue
+    }
+
+    var intValue: Int?
+
+    init?(intValue: Int) {
+        self.init(stringValue: "\(intValue)")
+        self.intValue = intValue
+    }
+}
+
+
+/// Add support do decode to a Dictionary
+extension KeyedDecodingContainer {
```

Comment:
How about adding a link to where this originated from?

Reply:
Done!